### ISSUE TITLE
Version 0.1.3 - stuck in Loading services infinitely

### ISSUE THREAD

**Issue by nhippe-ds** [grafana/explore-logs#738]
Running a local Grafana instance via docker-compose and the latest version (0.1.3) doesn't load services.

0.1.2 working:
![Image](https://github.com/user-attachments/assets/e4b9c9a2-9903-4533-9bbb-a0a5f3756e1d)

0.1.3 broken:
![Image](https://github.com/user-attachments/assets/7ea0f403-02e7-4cb5-8ac4-9aa9bc6ae82d)

Logs (changed to DEBUG) didn't show much:
```
024-08-28 08:13:19 logger=context userId=1 orgId=1 uname=admin t=2024-08-28T13:13:19.492839Z level=info msg="Request Completed" method=GET path=/api/live/ws status=-1 remote_addr=192.168.65.1 time_ms=3 duration=3.912167ms size=0 referer= handler=/api/live/ws status_source=server
2024-08-28 08:13:19 logger=live t=2024-08-28T13:13:19.507717208Z level=debug msg="Client connected" user=1 client=58dbd028-1004-4143-97ac-b8cac1775a07
2024-08-28 08:13:19 logger=accesscontrol t=2024-08-28T13:13:19.511241791Z level=debug msg="Evaluating permissions" namespace=user id=1 orgID=1 permissions="action:plugins.app:access scopes:plugins:id:grafana-lokiexplore-app"
2024-08-28 08:13:19 logger=accesscontrol.evaluator t=2024-08-28T13:13:19.511286875Z level=debug msg="Matched scope" userscope=plugins:* targetscope=plugins:id:grafana-lokiexplore-app
2024-08-28 08:13:19 logger=accesscontrol t=2024-08-28T13:13:19.516445541Z level=debug msg="Evaluating permissions" namespace=user id=1 orgID=1 permissions="action:plugins.app:access scopes:plugins:id:grafana-lokiexplore-app"
2024-08-28 08:13:19 logger=accesscontrol.evaluator t=2024-08-28T13:13:19.516483041Z level=debug msg="Matched scope" userscope=plugins:* targetscope=plugins:id:grafana-lokiexplore-app
2024-08-28 08:13:20 logger=ngalert.scheduler t=2024-08-28T13:13:20.001357167Z level=debug msg="No changes detected. Skip updating"
2024-08-28 08:13:29 logger=context userId=1 orgId=1 uname=admin t=2024-08-28T13:13:29.495027379Z level=debug msg="Received unknown frontend metric" metric=frontend_awaited_plugins_preload_ms
2024-08-28 08:13:29 logger=context userId=1 orgId=1 uname=admin t=2024-08-28T13:13:29.495069254Z level=debug msg="Received unknown frontend metric" metric=frontend_app_init_ms
2024-08-28 08:13:29 logger=context userId=1 orgId=1 uname=admin t=2024-08-28T13:13:29.495074088Z level=debug msg="Received unknown frontend metric" metric=frontend_plugin_preload_grafana-lokiexplore-app_ms
2024-08-28 08:13:30 logger=ngalert.scheduler t=2024-08-28T13:13:30.002014713Z level=debug msg="No changes detected. Skip updating"
```

docker-compose.yaml
```
version: '3.8'

services:
  grafana:
    image: grafana/grafana:latest
    container_name: grafana
    ports:
      - '3000:3000'
    networks:
      - monitoring
    environment:
      GF_SECURITY_ADMIN_USER: admin
      GF_SECURITY_ADMIN_PASSWORD: admin
      GF_INSTALL_PLUGINS: "https://storage.googleapis.com/integration-artifacts/grafana-lokiexplore-app/grafana-lokiexplore-app-0.1.2.zip;grafana-lokiexplore-app"
    volumes:
      - grafana_data:/var/lib/grafana
      - ./datasources.yaml:/etc/grafana/provisioning/datasources/datasources.yaml
```

Loki details:
* version 3.1.0
* all prerequisites set (pattern-ingester enabled, volume_enabled - 0.1.2 _does_ work)

**Comment by nhippe-ds** [comment]
appears to have been a local issue with Docker - now resolved - closing

**Comment by bytaj** [comment]
Did you do something to resolve it or did you modify something? I'm having the same issue @nhippe-ds

**Comment by dmitrievav** [comment]
same problem on EKS, with prometheus-operator helm chart deployment.
fixed by rolling back to version 0.1.2

```
# helm values
grafana:
   plugins:
           # https://github.com/grafana/explore-logs/releases
           # - grafana-lokiexplore-app
           - https://grafana.com/api/plugins/grafana-lokiexplore-app/versions/0.1.2/download;grafana-lokiexplore-app
```
